backuprestore.adoc

```diff
@@ -800,12 +799,14 @@ value.
 
 Three types of restore procedures can be run:
 
-The `zmrestore` command is used to restore the mailboxes while the
+* The `zmrestore` command is used to restore the mailboxes while the
 {product-name} mailbox server is running.
 
-* `zmrestoreoffline` -- to restore the mail server when the mail server is
-down. This command is run for disaster recovery.
-* `zmrestoreldap` -- to restore the content of the LDAP directory server.
+* The `zmrestoreoffline` command is used restore the mail server when
+the mail server is stopped. This command is run for disaster recovery.
```

Comment:
Again, doesn't remove the ambiguity as to what mail server stopped means.  Suggest link as suggested above, unless we can cut down on repetition - why does zmrestoreoffline need to appear in 2 lists?